build.sbt

```diff
@@ -11,20 +11,32 @@ skip in publish := true
 lazy val fetch = crossProject(JSPlatform, JVMPlatform)
   .crossType(CrossType.Pure)
   .settings(commonCrossDependencies)
+  .settings(
+    scalaVersion := "3.0.0-M3",
+    crossScalaVersions := Seq("2.12.11", "2.13.2", "3.0.0-M3")
```

Comment:
I guess we could use the latest versions for Scala 2.12 and 2.13:
- 2.12.13
- 2.13.5